pyp2spec/template.spec

```diff
@@ -12,7 +12,12 @@ Source:         {{source}}
 BuildArch:      noarch
 {%- endif %}
 BuildRequires:  python3-devel
```

Comment:
Juts a thought: What if python3-devel was part of ADDITIONAL_BUILD_REQUIRES_ARCHFUL and ADDITIONAL_BUILD_REQUIRES_NOARCH, so the template would not need to hardcode it? (Not a blocker for this PR, just brainstorming out loud.)

Reply:
Thinking about what's given and what's variable, `python3-devel` feels more on the "given" side, like a lot of other spec boilerplate.
OTOH, it IS an additional build requirement and the current situation forces a user who asks "what BR's will be added to my specfile" to realize that each of the BR's come from a different place in the codebase. Not optimal.
I'll let this idea sink for a moment.